tests/utils/UserTokenStoreMock.ts

```diff
@@ -0,0 +1,61 @@
+import { Octokit } from "@octokit/core";
+import { PaginateInterface } from "@octokit/plugin-paginate-rest";
+import { RestEndpointMethods } from "@octokit/plugin-rest-endpoint-methods/dist-types/generated/method-types";
+import { Api } from "@octokit/plugin-rest-endpoint-methods/dist-types/types";
+import { GitLabClient } from "../../src/Gitlab/Client";
+import { JiraClient } from "../../src/Jira/Client";
+import { JiraOAuth } from "../../src/Jira/OAuth";
+import { UserTokenStore } from "../../src/UserTokenStore";
+
+export class UserTokenStoreMock {
+    public jiraOAuth?: JiraOAuth;
+
+    public load(): Promise<void> {
```

Comment:
Any reason why all of these are defined at all (since they throw anyway)?

Seems like this entire Mock is actually unused (see the GithubRepoTest suggestion).